src/include/serial_inline.h

```diff
@@ -29,7 +30,15 @@ __insert_simple_func(
      * implementations read the old value multiple times.
      */
     for (i = 0; i < skipdepth; i++) {
-        WT_INSERT *old_ins = *ins_stack[i];
+        /*
+         * The insert stack position must be read only once - if the compiler chooses to re-read the
```

Comment:
If you choose my suggested re-wording of the comment above, you can also use similar words here for consistency.